src/nifgen/system_tests/test_system_nifgen.py

```diff
@@ -50,3 +50,30 @@ def test_get_self_cal_last_date_and_time():
 
 def test_self_cal(session):
         session.self_cal()
+
+
+def test_standard_waveform(session):
+    session.output_mode = nifgen.OutputMode.NIFGEN_VAL_OUTPUT_FUNC
+    session.configure_standard_waveform(nifgen.Waveform.SINE, 2.0, 0.0, 0.0, 2000000)
```

Comment:
You should assert everything you set in this function below.
You should avoid using repeated values so you are sure the right parameter made it to the right thing.
What are you trying to accomplish with a start phase of 2 million degrees?
You should also test that some values have defaults.